### ISSUE TITLE
Resolve "qs" package security vulnerability: port to v4.16.0 of Express

### ISSUE THREAD

**Issue by mihirsamdarshi** [dondi/GRNsight#564]
One of our packages in package-lock.json depends on a version of qs that is less than 1.0.0. This dependency has a security vulnerability. We should upgrade the package which depends on qs so we don't have this security vulnerability any more.

**Comment by mihirsamdarshi** [comment]
Express is the package, I am now looking into upgrading Express

**Comment by mihirsamdarshi** [comment]
Getting error `Error: Most middleware (like logger) is no longer bundled with Express and must be installed separately. Please see https://github.com/senchalabs/connect#middleware.` Will try to see what it takes to resolve this

**Comment by kdahlquist** [comment]
@mihirsamdarshi will type up what he reported in the meeting.  @eileenchoe said she would help since she did this last semester.

**Comment by mihirsamdarshi** [comment]
Currently, in order to upgrade 3.3.8 to 4.16.0, it will take changing the app.js in both server and web-client, and upgrading the various dependencies and middlewares that now must be installed and declared separately.

**Comment by mihirsamdarshi** [comment]
No transition from jade to pug is needed, although it might be helpful. Express has been updated to @4.16.0, and now all dependencies and packages have been updated. Pull request #565 has been created to merge into beta.

**Comment by kdahlquist** [comment]
Has the wiki been updated as well?

**Comment by mihirsamdarshi** [comment]
No, I am waiting on the pr to be incorporated before I update the wiki

**Comment by eileenchoe** [comment]
Approved PR #565 & refreshed beta

**Comment by kdahlquist** [comment]
Since PR was accepted and beta refreshed, this is closable.
